exonum/src/node/consensus.rs

```diff
@@ -500,6 +498,9 @@ impl NodeHandler {
                 .new_height(&block_hash, self.system_state.current_time());
             (block_state.txs().len(), block_state.proposer_id())
         };
+
+        self.api_state.broadcast(&block_hash);
```

Comment:
Just to clarify. Do I understand right that `.broadcast` executes asynchronously ?

Reply:
That's right

Reply:
I better prefer `notify` rather than `broadcast`

And why we didn't sending some "block info" instead of `block hash`, that's how we can remove `blockchain` dependency in `Actor` and make it more independent.